wrappers/golang/pqc/ml-kem/kyber_params.go

```diff
@@ -0,0 +1,93 @@
+package mlkem
+
+// #cgo CFLAGS: -I./include/
+// #include "mlkem.h"
+import "C"
+
+import "github.com/ingonyama-zk/icicle/v3/wrappers/golang/runtime"
+
+const (
+	ENTROPY_BYTES = 64
+	MESSAGE_BYTES = 32
+)
+
+type KyberParams struct {
+	PublicKeyBytes    int
+	SecretKeyBytes    int
+	CiphertextBytes   int
+	SharedSecretBytes int
+	K                 uint8
+	Eta1              uint8
+	Eta2              uint8
+	Du                uint8
+	Dv                uint8
```

Comment:
These params aren't used anywhere and aren't passed across the FFI. I think we can remove them?